root/static/styles/layout.less

```diff
@@ -245,9 +245,9 @@ pre code {
                 float: left;
                 list-style-type: none;
                 list-style-image: none;
-                &:hover {
+                &:hover, &:active, &.fake-active {
                     background-color: @musicbrainz-orange;
-                    > .menu-header {
+                    > a, .menu-header {
```

Comment:
I added it back for log in and create account - it feels weird to set those as class menu header because they're not the headers for a menu.